src/backend/planner/seq_scan_plan.cpp

```diff
@@ -229,7 +229,7 @@ int SeqScanPlan::SerializeSize() {
         size = size + Predicate()->SerializeSize();
     }
     if (Parent()) {
-        size = size + Parent()->SerializeSize();
```

Comment:
1. The Parent() and Predicate() method are duplicated in AbstractPlan. Please get rid of them.
2. For null checks, please use the form `if (ptr != nullptr)`